src/test/java/testng/BooleanValueIsNegative.java

```diff
@@ -0,0 +1,12 @@
+package testng;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class BooleanValueIsNegative extends BaseTest {
+    @Test(groups = {"expanded"})
+    public void testValueIsNegative() {
+        System.out.println("Thread id is: " + Thread.currentThread().getId() + " ");
```

Comment:
You could move this line to @before method and it will be triggered before each test.